assignment1/isPermutation.py

```diff
@@ -0,0 +1,28 @@
+def isPermutation(str1, str2):
```

Comment:
Add a docstring describing what the function does, what its arguments are, and what it returns.